src/main.js

```diff
@@ -35,112 +35,118 @@ startButton.addEventListener('click', startCountDown);
-        button1.classList.add(category1);
-    } else if (button3 == undefined) {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-    } else {
-        button1.classList.add(category1);
-        button2.classList.remove(category2);
-        button3.classList.remove(category3);
-    }
+function addClassProperty(button1, category1, button2, category2) {
+  if (!button2) {
+    button1.classList.add(category1);
+  } else {
+    button1.classList.add(category1);
+    button2.classList.add(category2);
+  }
+}
+
+function removeClassProperty(button1, category1, button2, category2) {
+  if (!button2) {
+    button1.classList.remove(category1);
+  } else {
+    button1.classList.remove(category1);
+    button2.classList.remove(category2);
+  }
 }
 
 function changeStudyColor() {
-    changeColor(studyButton, 'study-active', meditateButton, 'meditate-active', exerciseButton, 'exercise-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(studyButton, 'study-active');
+  removeClassProperty(meditateButton, 'meditate-active', exerciseButton, 'exercise-active')
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function changeMeditateColor() {
-    changeColor(meditateButton, 'meditate-active', studyButton, 'study-active', exerciseButton, 'exercise-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(meditateButton, 'meditate-active');
+  removeClassProperty(studyButton, 'study-active', exerciseButton, 'exercise-active');
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function changeExerciseColor() {
-    changeColor(exerciseButton, 'exercise-active', studyButton, 'study-active', meditateButton, 'meditate-active');
-    isCatChosen(studyButton, meditateButton, exerciseButton);
+  addClassProperty(exerciseButton, 'exercise-active');
+  removeClassProperty(studyButton, 'study-active', meditateButton, 'meditate-active');
+  isCatChosen(studyButton, meditateButton, exerciseButton);
   }
 
 function limitTimeInput(timeInput, num) {
   preventInvalids(event, timeInput);
   timeInput.value = parseInt(timeInput.value);
   if (timeInput.value > num) {
-      timeInput.value = num;
+    timeInput.value = num;
   }
 }
 
 function preventInvalids(event, inputField) {
-    var invalidChars = ['+', '-', 'e', 'E'];
-    for (var i = 0; i < invalidChars.length; i++) {
-        if (event.key === invalidChars[i]) {
-            event.preventDefault();
-            inputField.value = '';
-        }
+  var invalidChars = ['+', '-', 'e', 'E'];
+  for (var i = 0; i < invalidChars.length; i++) {
+    if (event.key === invalidChars[i]) {
+      event.preventDefault();
+      inputField.value = '';
     }
+  }
+}
+function updateErrorMsg(userInput, errorMsgIndex) {
+  userInput.value ? 
+  addClassProperty(errorMsg[errorMsgIndex], 'hidden') : removeClassProperty(errorMsg[errorMsgIndex], 'hidden');
```

Comment:
nice use of ternary, its great for things that you can put on one line and are easily understandable.